Podfile

```diff
@@ -5,28 +5,24 @@ inhibit_all_warnings!
 def testing_pods
   pod 'Quick'
   pod 'Nimble'
-  pod 'RxNimble', subspecs: ['RxBlocking', 'RxTest']
-  pod 'RxSwift'
   # TODO: Remove or update the version of `1.8.0` to the newest version (not 1.8.1) when init a new project.
   # Currently, there is a bug on `1.8.1` - the newest version.
   pod 'Sourcery', '1.8.0'
   pod 'SwiftFormat/CLI'
 end
 
-target '{PROJECT_NAME}' do
+target 'Survey' do
   # UI
   pod 'Kingfisher'
   pod 'SnapKit'
 
-  # Rx
-  pod 'RxAlamofire'
-  pod 'RxCocoa'
-  pod 'RxDataSources'
-  pod 'RxSwift'
-
   # Storage
   pod 'KeychainAccess'
 
+  # Networking
+  pod 'Alamofire'
+  pod 'Japx'
```

Comment:
Could you provide usecase for `Japx`? Currently we use Swift's `JSONDecoder` (as you see in the template). What does Japx improve for the project that JSONDecoder is missing? 🙏

Reply:
Since REST API

Reply:
I used `Japx` because the application follows the standard [JSON API spec ](https://jsonapi.org/) v1 and it's recommended for client app to use a third-party to deserialize JSON API responses with ease.
According to https://github.com/nimblehq/nimble-survey-web/wiki/REST-API

Reply:
I used Japx in my Nimble Survey challenge too, it help to flatmap  the json properties. I think this is Ok